metriken-exposition/src/lib.rs

```diff
@@ -3,8 +3,6 @@
 //! Provides a standardized struct for a snapshot of the metric readings as well
 //! as a way of producing the snapshots.
 
-pub use histogram::Snapshot as HistogramSnapshot;
```

Comment:
Do we want to expose histogram with something like `pub use histogram::Histogram as Histogram` or not necessary?

Reply:
While it would be more convenient, we already have a public type called Histogram which is the histogram with name and metadata. 

I think in a future (and breaking) version - we may wish the make the fields of Counter, Gauge, and Histogram private and provide indirection instead.